### PR TITLE
Update with script example in the right page

### PR DESCRIPTION
**Pull request by cloudmelon**
Update with script example in the right page

## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVmssExtension.md [hunk 1]
```diff
@@ -30,6 +30,24 @@ The **Remove-AzureRmVmssExtension** cmdlet removes an extension from the Virtual
...
+PS C:\> Remove-AzureRmVmssExtension -VirtualMachineScaleSet $vmss -Name $vmssExtensionName
+PS C:\> Update-AzureRmVmss -ResourceGroupName $RGName -Name $vmssName -VirtualMachineScaleSet $vmss
+```
+
+This command removes the extension of a VMSS and update the VMSS after the modification.
+
+### Example 2: Remove an instance from within a VMSS
+```
+PS C:\> $vmss = Get-AzureRmVmss -ResourceGroupName $RGName -VMScaleSetName $vmssName 
+PS C:\> Remove-AzureRmVmss -ResourceGroupName "Group002" -VirtualMachineScaleSet $vmss -Id $extensionId
```
Comment (cormacpayne) [inline src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVmssExtension.md]: @cloudmelon was this meant to be `Remove-AzureRmVmssExtension`?

File: src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVmssExtension.md [hunk 2]
```diff
@@ -30,6 +30,24 @@ The **Remove-AzureRmVmssExtension** cmdlet removes an extension from the Virtual
 
 ## EXAMPLES
 
+### Example 1: Remove a VMSS extension
+```
+PS C:\> $vmss = Get-AzureRmVmss -ResourceGroupName $RGName -VMScaleSetName $vmssName 
+PS C:\> Remove-AzureRmVmssExtension -VirtualMachineScaleSet $vmss -Name $vmssExtensionName
```
Comment (cormacpayne) [inline src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVmssExtension.md]: @cloudmelon the `-ResourceGroupName` parameter is mandatory for this cmdlet, so you'll need to make sure to include it

File: src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVmssExtension.md (hunk 1 above)
Comment (cloudmelon) [reply src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVmssExtension.md]: Yes. Thanks @cormacpayne  for all your patience :)

File: src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVmssExtension.md (hunk 2 above)
Comment (cloudmelon) [reply src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVmssExtension.md]: https://docs.microsoft.com/en-us/powershell/module/azurerm.compute/remove-azurermvmssextension?view=azurermps-6.4.0

Actually the resource group name is used in 1st command to get the vmss as a object then we'll use it in passing as object parametter in Remove-AzureRmVmssExtension so it is not mandatory as mentionned in the documentation.  Thanks for verifying it with me :)

Comment (cormacpayne) [reply src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVmssExtension.md]: @cloudmelon whoops, was looking at the documentation for `Remove-AzureRmVmss`, which has a required `-ResourceGroupName` parameter 😁

Comment (cloudmelon) [reply src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVmssExtension.md]: yes @cormacpayne that is for Remove-AzureRmVmss, this page is about Remove-AzureRmVmssExtension :)

**Pull request merged**